src/main/resources/swagger/api.yml

```diff
@@ -508,6 +508,9 @@ definitions:
             "path":
                 type: string
                 description: "Json pointer, e.g. '/characteristic/-'"
+            "reason":
+              type: string
+              description: "Operation reason"
```

Comment:
let's extand model according to TMF recomnadation (for example how it was done on ms communication)

Reply:
@vkirichenko May you give an example on ms communication?

Reply:
https://github.com/xm-online/tmf-ms-communication/blob/master/src/main/resources/swagger/api-extension.yml

Reply:
@vkirichenko extension.yml(-s) was added